structuredefinitions/Extension-England-RemovalReason.xml

```diff
@@ -0,0 +1,61 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-RemovalReason" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-RemovalReason" />
```

Comment:
Need to be https://fhir.nhs.uk/StructureDefinition/Extension-England-FGMRemovalReason

Reply:
Agree. Please modify